.travis.yml

```diff
@@ -2,15 +2,26 @@ language: r
 sudo: false
 cache: packages
 
+r:
+  - oldrel
+  - release
+  - devel
+
+env:
+  - DEPS=github
+  - DEPS=cran
+
+r_packages:
+  - devtools
+
 r_github_packages:
-  - IRkernel/repr
+  # we currently still need IRdisplay from github in all cases...
   - IRkernel/IRdisplay
 
-addons:
-  apt:
-    packages:
-      - python3
-      - python3-setuptools
 
 script:
   - source .travis-build.sh
```

Comment:
Hm, having it executed would be better because than you could just `#! bash -e` instead of the `set -e ... set +e` dance..

There is BTW one big reason to put it all into travis.yml: the commands are echo'ed and you see what command had the error :-)